content/introduction/overview.md

```diff
@@ -13,15 +13,19 @@ What does that mean, exactly? Let’s say you’re doing some research on Aardva
 https://en.wikipedia.org/wiki/Aardvark
 ```
 
-When you put that URL in your browser’s address bar, your computer asks one of Wikipedia’s computers, which might be somewhere on the other side of the country — or even the planet — for the Aardvark page. However, if you use IPFS to get that page from:
+When you put that URL in your browser’s address bar, your computer asks one of Wikipedia’s computers, which might be somewhere on the other side of the country (or even the planet), for the Aardvark page.
```

Comment:
lowercase "aardvark" throughout?

Reply:
Shrug. I left it the way I found it, but don't have strong opinions. I shall lowercase!